substrate-archive/src/actors/workers/database.rs

```diff
@@ -164,11 +164,12 @@ impl<B: BlockT> Handler<Storage<B>> for DatabaseActor<B> {
 #[async_trait::async_trait]
 impl<B: BlockT> Handler<BatchStorage<B>> for DatabaseActor<B> {
 	async fn handle(&mut self, storages: BatchStorage<B>, _ctx: &mut Context<Self>) {
+		let len = storages.inner.iter().map(|storage| storage.changes.len()).sum::<usize>();
 		let now = std::time::Instant::now();
 		if let Err(e) = self.batch_storage_handler(storages).await {
 			log::error!("{}", e.to_string());
 		}
-		log::debug!("took {:?} to insert storage", now.elapsed());
+		log::debug!("Took {:?} to insert {} storages", now.elapsed(), len);
```

Comment:
```suggestion
		log::debug!("Took {:?} to insert {} storage entries", now.elapsed(), len);
```

small nit, `storages` just sounds odd